<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(112, 112, 112);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="line_name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>NAME</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_pass">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>PASSWORD</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_error">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_btn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password anzeigen</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>241</width>
      <height>27</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="btn_login">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 0);</string>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_exit">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 85, 255);</string>
     </property>
     <property name="text">
      <string>EXIT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clear">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 255);</string>
     </property>
     <property name="text">
      <string>CLEAR</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_combi">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(11, 15, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_easter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="Ressource.qrc">:/easter.webp</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>496</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/icons8-abschalttaste-50.png</normaloff>:/icons8-abschalttaste-50.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Ressource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
